Library/Homebrew/dev-cmd/pr-upload.rb

```diff
@@ -48,12 +48,36 @@ def check_bottled_formulae(json_files)
   def pr_upload
     args = pr_upload_args.parse
 
-    bintray_org = args.bintray_org || "homebrew"
-    bintray = Bintray.new(org: bintray_org)
-
     json_files = Dir["*.json"]
     odie "No JSON files found in the current working directory" if json_files.empty?
 
+    bottles_hash = json_files.reduce({}) do |hash, json_file|
+      hash.deep_merge(JSON.parse(IO.read(json_file)))
+    end
+
+    bottles_hash.each do |_, bottle_hash|
+      root_url = bottle_hash["bottle"]["root_url"]
+      url_match = root_url.match HOMEBREW_RELEASES_URL_REGEX
+      _, user, repo, tag = *url_match
+
+      next unless tag
+
+      release = begin
```

Comment:
Should this be conditional on a `--github-release` argument or something or is the above code doing just something smart based on the `root_url`? If so: I think this probably warrants a comment above this loop that this whole thing is purely to handle uploading to GitHub releases. Also: if we do that do we want to skip uploading to Bintray?

Reply:
Yes, `root_url` is being matched as GitHub Releases url, if it is matched then we upload and should skip Bintray stuff entirely.

Reply:
I'm not sure if someone ever will end up having bottles to upload with different `root_url`s.

Reply:
Cool, fine with me 👍🏻 I'd say:
- add a comment
- `return` to avoid duplicate Bintray upload